src/modules/auth/auth.service.ts

```diff
@@ -211,9 +220,17 @@ export class AuthService {
     }
 
     await this.usersService.confirmUser(user.id);
+
     const publicUser = this.usersService.buildPublicUser(user);
-    const userWithTokens = await this.generateUserWithTokens(publicUser);
+    const verifiedUser = this.makeUserVerified(publicUser);
+    const userWithTokens = await this.generateUserWithTokens(verifiedUser);
 
     return userWithTokens;
   }
+
+  async changePassword(id: string, dto: PasswordDto): Promise<void> {
```

Comment:
its service why you pass full dto if only password used?

Reply:
and name dto does not mean noting, naming should be self explained

Reply:
For consistency across the project, I always pass the DTO from the controller to the service and destructure the DTO into variables within the service. The current PasswordDto contains only one field - {"password": "..."}. It is used for validating incoming data using class-validator through UsePipes. This approach is also consistent throughout our project, ensuring that any incoming JSON that fails validation shows an error to the user and does not proceed to the controller or to the service.

I will change names of variables.